src/utils/DefaultMap.spec.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2023 Google LLC.
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as chai from 'chai';
+
+import {DefaultMap} from './DefaultMap.js';
+
+const expect = chai.expect;
+
+describe.only('DefaultMap', () => {
+  it('returns the default value when key does not exist', () => {
+    const defaultValue = 0;
+
+    const cutenessMap = new DefaultMap<string, number>(
+      () => defaultValue,
+      [['dog', 100]]
+    );
+
+    expect(cutenessMap.get('dog')).to.deep.equal(100);
+    expect(cutenessMap.get('cat')).to.deep.equal(defaultValue);
```

Comment:
Also we should test `.values()` contains `cat` after accessing it. We check that we don't return a new default each time.

Reply:
Done...I think(?) You probably meant `keys()`, not `values()`.